### DISCUSSION TITLE
Is there a way to use HTML tags in descs?

### DISCUSSION THREAD

**Discussion by boberski666** [mediacms-io/mediacms#847] [Q&A]
Is there a way to use HTML tags in descs?

**Comment by boberski666** [comment]
I saw some props for that but changing them do not work

**Comment by boberski666** [comment]
@mgogoulos can you give a hint on how to do it?

**Comment by mgogoulos** [comment]
@MBoberski sorry for replying late to this, basically all you have to do is edit files.models.py and remove 'description' from line

```
strip_text_items = ["title", "description"]
```

on save function of model Media. This is responsible for stripping all html from description. If you remove this and restart the services, then  the frontend app will show the text as html. 

Try it out and let me know if it works for you! And if you want please add an entry on the documentation (on the admin.docs)
